docs/mongodb.md

```diff
@@ -0,0 +1,37 @@
+# MongoDB
+
+## 5.0 to 7.0
+These instructions allow developers to move from MongoDB 5.0 to 7.0 to match the database provided by Database Services. As re-ingestion only takes a few minutes, completely wiping the machine of 5.0 and doing a clean install for 7.0 seems like an easier alterative than upgrading from 5.0 to 6.0 and then 6.0 to 7.0. These commands have been tested on a Rocky 8 VM but should work on Rocky 9 providing the small edit is made to the Yum repo
+
+```bash
+# Stop MongoDB process
+systemctl stop mongod
+# Uninstall packages using yum
+yum erase mongodb-*
+# Check that everything's been uninstalled
+yum list installed | grep -i mongod
+# Remove data directories, databases, logs
+rm -rf /var/log/mongodb /var/lib/mongo*
+# Disable yum repo for MongoDB 5.0
+yum-config-manager --disable mongodb-org-5.0
+
+# Add yum repo for MongoDB 7.0
+touch /etc/yum.repos.d/mongodb-org-7.0.repo
+
+# Paste the following into /etc/yum.repos.d/mongodb-org-7.0.repo
+# Edit the /8/ to /9/ if you're using Rocky 9
+[mongodb-org-7.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/7.0/x86_64/
+gpgcheck=1
+enabled=1
+gpgkey=https://pgp.mongodb.com/server-7.0.asc
```

Comment:
Since this isn't a bash command I think it would be better to make it a separate code block, e.g (formatting isn't great because of the nested backticks but hopefully it makes sense:
```suggestion

Paste the following into /etc/yum.repos.d/mongodb-org-7.0.repo (edit the /8/ to /9/ if you're using Rocky 9):
```ini
[mongodb-org-7.0]
name=MongoDB Repository
baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/7.0/x86_64/
gpgcheck=1
enabled=1
gpgkey=https://pgp.mongodb.com/server-7.0.asc
```

```bash
```

Reply:
I've committed all the suggested changes you proposed, thanks for making them. I had to play around with the formatting on this suggestion and settled on the changes in [f82776c](https://github.com/ral-facilities/operationsgateway-api/pull/118/commits/f82776cee9023178576dc8dee6d3fba0568dd322). Can you take a look and make sure you're happy with how it looks please?